src/index.ts

```diff
@@ -304,14 +304,17 @@ function math_block_dollar(
         start = state.bMarks[nextLine] + state.tShift[nextLine]
         end = state.eMarks[nextLine]
         if (end - start < 2) {
-          continue
+          break // blank lines are not allowed within $$
```

Comment:
This logic is wrong, it fails on lines consisting of a single character. I think I assumed that \n was counted in the total.

Reply:
That does fix the tests. I will push and then get a sanity check from @chrisjsewell.

Reply:
Any update on this? I'd like to port a documentation tool for the Lean programming language from using `mistletoe` to useing markdown-it, but can't do so without this fix!

Reply:
I haven't heard from @chrisjsewell in the last week. I did ping him on slack. I unfortunately don't have npm admin rights to update this at the moment.